src/main/java/edu/oregonstate/cs361/battleship/Ship.java

```diff
@@ -2,23 +2,22 @@
 
 import static edu.oregonstate.cs361.battleship.BattleshipModel.GRID_SIZE;
 
-class Ship {
+public class Ship {
```

Comment:
Does making this public make a difference, considering all of these classes belong to the same package?